src/content/blog/astro-560.mdx

```diff
@@ -0,0 +1,274 @@
+authors:
+  - matt
+  - ema
+coverImage: '/src/content/blog/_images/astro-560/blog-post-5-6.webp'
+socialImage: '/src/content/blog/_images/astro-560/og-astro-5-6.webp'
+lang: 'en'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+import Note from '~/components/Note.astro';
+import Mention from '/src/components/Mention.astro';
+
+**Astro 5.6 brings first-class astro:env and experimental session support to Cloudflare, and gives more control over prefetching.**
+
+☁️ Soar through the clouds with these new features in Astro:
+
+- [**Global `astro:env` on Cloudflare**](#global-astroenv-on-cloudflare)
+- [**Experimental sessions on Cloudflare**](#experimental-sessions-on-cloudflare)
+- [**New prefetch eagerness option**](#new-prefetch-eagerness-option)
+- [**Custom fetch option for prerendered error pages**](#custom-fetch-option-for-prerendered-error-pages)
+- [**New `load()` method for experimental sessions**](#new-load-method-for-experimental-sessions)
+- [**Improved config validation**](#improved-config-validation)
+- [**Breaking changes to the experimental SVG API**](#breaking-changes-to-the-experimental-svg-api)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Global `astro:env` on Cloudflare
+
+Since Astro 5 you have been able to access [type-safe environment variables using `astro:env`](https://docs.astro.build/en/guides/environment-variables/#type-safe-environment-variables). However, environment variables in Cloudflare were only accessible within a request — limiting when and where you could use them. For example, if you wanted to use a shared API client instance, you couldn't use environment variables if instantiating outside of a request. This could be confusing and was different from our other adapters.
+
+Astro 5.6 lifts this restriction, meaning you can now access your environment variables globally throughout your server code and brings the Cloudflare adapter in line with the other official adapters. This is possible thanks to [enhancements in Cloudflare Workers](https://developers.cloudflare.com/changelog/2025-03-17-importable-env/) that are now implemented in Astro.
+
+```js
+import { defineMiddleware } from 'astro:middleware';
+import { API_URL } from 'astro:env/server';
+import { createClient } from './client.js';
+
+// Astro 5.5: undefined
+// Astro 5.6: string
+const client = createClient(API_URL);
+
+export const onRequest = defineMiddleware((ctx, next) => {
+	ctx.locals.client = client;
+	return next();
+});
+```
+
+This improvement creates a more consistent developer experience across all Astro server environments, and means you don't need to waste headspace working out what counts as request scope or global scope. 
```

Comment:
```suggestion
This improvement creates a more consistent developer experience across all Astro server environments, and means you don't need to worry about figuring out what counts as request scope or global scope. 
```
Nice how this all worked out and gave us something to put here! Just "uncolonizing" the language a bit, maybe?

Reply:
It means you sound very British! 😂